game/src/edit/roads.rs

```diff
@@ -1,14 +1,15 @@
-use geom::{CornerRadii, Distance, UnitFmt};
```

Comment:
Wow this is quickly replacing the traffic signal editor as one of our most complicated pieces of UI code. :)  No specific complaints, just lots of state to understand. All the helper functions are a good organization.